src/music/title-theme.zig

```diff
@@ -0,0 +1,55 @@
+// Placeholder implementation - not complete
+const tone = @import("tone.zig");
```

Comment:
You forgot to add this file to git

Reply:
Ah bother

Reply:
Will do in the morning

Reply:
Just realised - this file already exists at main.

Reply:
You happy for me to merge?